platforms/posix/src/px4/common/tasks.cpp

```diff
@@ -114,13 +107,13 @@ static void *entry_adapter(void *ptr)
 px4_task_t px4_task_spawn_cmd(const char *name, int scheduler, int priority, int stack_size, px4_main_t entry,
 			      char *const argv[])
 {
-
-	int i;
 	int argc = 0;
-	unsigned int len = 0;
 	struct sched_param param = {};
 	char *p = (char *)argv;
 
+	// The name is added as the first argument.
+	unsigned len = strlen(name) + 1;
```

Comment:
I don't see `name` being copied into argv of `taskdata`.

Reply:
Sorry, I think that's a leftover and should not be here. Let me check the diff again.